app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,52 @@
+class CommentsController < ApplicationController
+  load_and_authorize_resource
+
+  # issue_comments POST /issues/:issue_id/comments(.:format) comments#create
+  def create
+    @resource = Issue.find(params[:issue_id])
+    authorize! :read, @resource
+
+    @comment.commentable = @resource
+    @comment = Comment.new if @comment.save
+    @pagy, @comments = pagy(@resource.comments)
+    # TODO: Add a proper decorator class for this type of thing
+    @pagy.instance_variable_set(:@custom_link, project_issue_path(@resource.project, @resource))
+    respond_to :js
+  end
+
+  # edit_comment GET /comments/:id/edit(.:format) comments#edit
+  def edit
+    respond_to :js
```

Comment:
respond_to do |format|
  format.js
end

Reply:
This is like the shortcut to the same thing

Reply:
added the block